services/rule_engine/src/main/java/agents/Moli.java

```diff
@@ -44,17 +42,7 @@ public String getArmoryUrl() {
     }
 
     public String getIntroEmailText() {
-        return "Bok, \n"
-        + "ja sam Moli, miočanski AI asistent. \n" 
-        + "Danas provodim jednu malu anketu o Miocu i puno bi mi značila tvoja pomoć."
-        + "Neće ti uzeti puno vremena, ali meni će puno pomoći da vidim kako poboljšati Mioc. "
-        + "Plus, pomoći ćeš i sebi i svojim kolegama!\n\n"
-        + "Sve što za sad moraš učiniti je kliknuti na link koji vodi na anketu "
-        + "i ja ću te provesti kroz proces. Možeš ju ispuniti kad god želiš, "
-        + "tvoje mi je mišljenje jako bitno.\n\n"
-        + getArmoryUrl() + "\n\n"
-        + "Hvala ti na vremenu, \n"
-        + "Moli";
+        return Settings.getInstance().ONBOARDING_EMAIL_TEMPLATE.replace("%ARMORY_URL%", getArmoryUrl());
```

Comment:
We have functions for this (`com.mindsmiths.sdk.utils.templating.Templating.recursiveRender()`), but this method is ok for now.